package.json

```diff
@@ -123,7 +123,7 @@
     "sass-loader": "^10.5.2",
     "shallow-equal": "^3.1.0",
     "shortid": "2.2.8",
-    "superdesk-ui-framework": "4.0.54",
+    "superdesk-ui-framework": "^4.0.56",
```

Comment:
would rather stick to exact version so when we update ui framework we have to run CI on it